Testreglar/251/Nett/nett-2.5.1a.json

```diff
@@ -0,0 +1,127 @@
+			"type": "jaNei",
+			"kilde": [],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.1"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Testsiden har ikke funksjonalitet som krever flerpunkt eller stibasert gjest."
+				}
+			}
+		},
+		{
+			"stegnr": "3.1",
+			"spm": "Hvilken funksjonalitet tester du?",
+			"ht": "<p>Beskriv funksjonaliteten, slik at det er mulig å identifisere den i ettertid. Hvis det gjelder flere funksjoner, registrerer du en og en.</p>",
+			"type": "tekst",
+			"kilde": [],
+			"label": "Funksjonalitet:",
+			"multilinje": true,
+			"ruting": {
+				"alle": {
+					"type": "gaaTil",
+					"steg": "3.2"
+				}
+			}
+		},
+		{
+			"stegnr": "3.2",
+			"spm": "Er den aktuelle flerpunkts- eller stibaserte gesten er bestemt av innholdsprodusenten?",
+			"ht": "<p> </p>\n<p>Kravet gjelder ikke for standard pekerbevegelser for å betjene brukeragenter eller operativsystemer. For eksempel</p>\n<ul>\n<li>Brukeragent: horisontal sveiping for å flytte gjennom nettlesers historie eller vertikal sveiping for å bla gjennom sidens innhold</li>\n<li>Operativsystem: bevegelser for å skifte mellom åpne apper, eller hente frem kontekstuelle menyer med hjelpemiddelteknologier når disse er aktivert</li>\n</ul>\n<p>Sjekk om flerpunkts bevegelsen eller stibasert gest er bestemt av innholdsprodusenten.</p>",
+			"type": "jaNei",
+			"kilde": [],
+			"ruting": {
+				"ja": {
+					"type": "gaaTil",
+					"steg": "3.3"
+				},
+				"nei": {
+					"type": "ikkjeForekomst",
+					"utfall": "Flerpunkt eller stibasert gjest er bestemt av brukeragenten."
+				}
+			}
+		},
+		{
+			"stegnr": "3.3",
+			"spm": "Er flerpunktsgest eller stibasert gest avgjørende?",
+			"ht": "<p>Dersom avgjørende elementer fjernes, vil det medføre en grunnleggende endring av informasjonen eller funksjonaliteten i innholdet, <strong>og </strong>informasjonen og funksjonaliteten vil ikke kunne gjenskapes på en annen måte som samsvarer med kravene.</p>\n<p>Eksempler er en signatur eller frihåndstegning og sveip for godkjenne (native apper).</p>",
```

Comment:
Nativ apper?   Viss det viser til native apper på mobil/nettbrett er det vell ikke relevant for en nett-testregel?